packages/p2p-v2/src/components/PageReturn/PageReturn.scss

```
@@ -0,0 +1,25 @@
+.p2p-v2-page-return {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 2.4rem 0;
+    font-size: 16px;
+    font-weight: 700;
+    width: 100%;
+
+    @include mobile {
+        margin: 0;
+        padding: 1.6rem;
+    }
+
+    &__button {
+        line-height: 0.8rem;
+        cursor: pointer;
+        border-radius: $BORDER_RADIUS;
```

```suggestion
        border-radius: $BORDER_RADIUS;
```
Just a question @ameerul-deriv, where could the `$BORDER_RADIUS` be coming from / declared? 🤔

---

ah i forgot to delete this, thanks for pointing this out 🙏